programl/task/graph_level_classification/README.md

```diff
@@ -0,0 +1,51 @@
+# Graph Level Classification
+
+Two subtasks are of particular interest: classifyapp a.k.a. poj104 and devmap a.k.a. heterogeneous device mapping.
+
+## Quickstart
+`python run.py --help` will print this help
+```
+Usage:
+   run.py [options]
+
+Options:
+    -h --help                       Show this screen.
+    --data_dir DATA_DIR             Directory(*) to of dataset. (*)=relative to repository root ProGraML/.
+                                        Will overwrite the per-dataset defaults if provided.
+
+    --log_dir LOG_DIR               Directory(*) to store logfiles and trained models relative to repository dir.
+                                        [default: programl/task/graph_level_classification/logs/unspecified]
```

Comment:
@ChrisCummins would you like to specify a default log dir here? it is relative to the repository dir.

Reply:
For the dataflow experiment I put `logs/` inside the root of the dataset, see: https://github.com/ChrisCummins/ProGraML/blob/development/programl/Documentation/DataflowDataset.md#directory-layout

I think it makes sense to do that with the classifyapp/devmap too